projects/aws-observability/aws-otel-collector/Makefile

```diff
@@ -1,6 +1,9 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
+GIT_HASH:=$(shell git -C $(BASE_DIRECTORY) rev-parse HEAD)
```

Comment:
We should avoid this, it's only going to be a headache later. Jonathan and I had to do something similar for autoscaler and I'm not super happy about it. Can we talk thru this some?

Reply:
Yes. Let me touch base with you guys tomorrow.

Reply:
Updated. We will avoid overwriting the GIT_HASH and HELM_TAG.